content/applications/websites/livechat.rst

```diff
@@ -73,10 +73,10 @@ live chat channel has been added as an operator by default.
    modal, adjust any information, as needed. Then, click :guilabel:`Save`, or click
    :guilabel:`Remove` to remove that operator from the channel.
 
-Click :guilabel:`Add` to reveal an :guilabel:`Add: Operators` modal.
+Click :guilabel:`Add` to reveal an :guilabel:`Add: Operators` pop-up window.
 
-In the modal, scroll to find the desired users, or enter their name in the :guilabel:`Search...`
-bar. Then, click the checkbox next to the users to be added, and click :guilabel:`Select`.
+In the pop-up window, scroll to find the desired users, or enter their name in the search bar. Then,
+tick the checkbox next to the users to be added, and click :guilabel:`Select`.
 
 New operators can be created and added to the list directly from this modal, as well, by clicking
```

Comment:
```suggestion
New operators can be created and added to the list directly from this pop-up window, as well, by clicking
```
opting for this phrasing because it was already used above (as opposed to modal). Just to keep things uniform throughout...